lib/bundle-helpers.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
+  reBundle(moduleName, bundles, rebundleConfigs) {
```

Comment:
Explain the options in a comment, particularly the second and third — I'm not sure what I'm looking at.